small-talk/components/censorMess.js

```diff
@@ -0,0 +1,11 @@
+import swearWords from './swearWords.js';
+
+export default function censor(input) {
+    const inputinput = input.toLowerCase(); // make sure check for all cases
+    let censored = inputinput;
+    for (let swearWord of swearWords) { // check for all swear words
+        const regex = new RegExp(`\\b${swearWord}\\b`, 'gi'); // used AI to match words and replace
+        censored = censored.replace(regex, '*'.repeat(swearWord.length)); // replaces vulgar language with ****
+    }
+    return censored;
+}
```

Comment:
I think this'll make every word lowercase right? It should keep the original case sensitivity if it doesn't get censored.

Reply:
Yes that was a good catch thank you